src/main/java/core/basesyntax/fruitshop/report/generator/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.fruitshop.report.generator;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String TITLE = "fruit,quantity\n";
+
+    @Override
+    public void generateReport(String toFile, Map<String, Integer> storage) {
```

Comment:
receive ready String content because it's recommended to keep open connection to file as short as possible.